purchase_last_price_info/models/product_product.py

```diff
@@ -53,7 +53,10 @@ def _compute_last_purchase_line_id(self):
     @api.depends("last_purchase_line_id")
     def _compute_last_purchase_line_id_info(self):
         for item in self:
-            item.last_purchase_price = item.last_purchase_line_id.price_unit
+            item.last_purchase_price = (
+                item.last_purchase_line_id.price_subtotal
+                / item.last_purchase_line_id.product_qty
```

Comment:
Becarefull we should take care of the case where item.last_purchase_line_id.product_qty is 0
The best is to change the domain of last_purchase_line_ids

```

    last_purchase_line_ids = fields.One2many(
        comodel_name="purchase.order.line",
        inverse_name="product_id",
        domain=[("state", "in", ["purchase", "done"]), ("product_qty", ">", 0)],
        string="Last Purchase Order Lines",
)

```